WebDevStudios/wds-block-starter
# Feature/40 wp scripts 14 1 0

Closes https://github.com/WebDevStudios/wds-block-starter/issues/40

Large bump from WP Scripts `12.1.1` to `14.1.0`.

Note: I added details to the README changelog, and recommend that this be considered a minor change.

### Screenshot
Editor view displaying block in editor
<img width="1111" alt="Screen Shot 2021-04-07 at 2 06 34 PM" src="https://user-images.githubusercontent.com/25035900/113913979-f70d2380-97aa-11eb-86b0-2f43dd8f3f8a.png">

Frontend view displaying block on the frontend
<img width="1099" alt="Screen Shot 2021-04-07 at 2 06 23 PM" src="https://user-images.githubusercontent.com/25035900/113913977-f70d2380-97aa-11eb-8942-bdd682e75b12.png">

### Steps to verify the solution
1. Run `npm install` 
2. Verify `@wordpress/scripts` is visible in the package-lock.json
3. Run `npm run build` and verify the plugin built correctly by inserting the demo block

### Other
- [NA] Is this issue accessible? (Section 508/WCAG 2.1AA)
- [NA] Does this pass CrossBrowserTesting.com? (Edge, Safari, Chrome, Firefox)
- [NA] Will this pull request require [updating the wiki?](https://github.com/WebDevStudios/wds-block-starter/wiki)
